core-services/prow/02_config/red-hat-storage/volume-replication-operator/_prowconfig.yaml

```diff
@@ -0,0 +1,29 @@
+tide:
+  queries:
+  - excludedBranches:
+    - devel
```

Comment:
it should be devel or main? for volume replication we use `main` branch we don't have `devel` branch

Reply:
fixed